### ISSUE TITLE
[Naxxramas] Gothik - Doppel-Spawn Reiter

### ISSUE THREAD

**Issue by Manixer** [ClassicWoW/Nefarian_1.12.1_Bugtracker#2636]
Welches Verhalten wird beobachtet?
Bei Gothik spawnen 2 Reiter was fast immer zu einem Raid-Wipe führt.

Wie sollte es sich verhalten?
Es sollte immer nur ein Reiter spawnen.

Schritte zur Reproduzierung
-

Zusätzliche Informationen (Screenshots, Videos, Klasse, Rasse, Level, etc.)
Video wird angefügt sobald verfügbar.

Alle Kreaturen, Items, Objekte, Quests, Zauber etc. müssen aus unserer Datenbank verlinkt sein.
https://datenbank.classic-wow.org/?npc=16060

**Comment by Haudraufinix** [comment]
Hier eine Quelle dazu ausm Dungeon companion version 2
https://imgur.com/a/H8HGH8g

**Comment by Dotox** [comment]
Wurde mir schon einmal irgendwann berichtet, allerdings konnte es nicht reproduziert werden.

**Comment by Suzaru** [comment]
Also bei der Legion haben wir es auch schon mehrfach gehabt. Ich höre mich mal um, ob es an Video gibt.
Leider habe ich dabei keine Korrelation erlebt, es schien vollkommen random zu sein.

**Comment by Sheed90** [comment]
Ja, diesen Bug gibt es, allerdings wird er nur selten bis gar nicht ausgelöst. Wir hatten ihn die letzten Wochen nicht. Bei uns trat er allerdings immer beim letzten Reiter auf, der spawnen sollte.

**Comment by Suzaru** [comment]
Bei uns heute wieder :) nur fürs Protokoll.
